megamek/src/megamek/common/equipment/ArmorType.java

```diff
@@ -391,7 +391,7 @@ private static ArmorType createClanFerroFibrous() {
         armor.addLookupName("Clan Ferro-Fibrous Armor");
         armor.addLookupName("Clan Ferro Fibre");
         armor.cost = 20000.0;
-        armor.criticals = 7;
+        armor.criticals = CRITICALS_VARIABLE;
```

Comment:
How does these changes impact non-superheavy units?

Reply:
They don't. Nothing accesses `MiscType.criticals` directly, it goes through the `getCriticals` method. `getCriticals` already had the correct CRITICALS_VARIABLE handler for all armor types to report the number of crits to be the standard value in all cases except for superheavy meks, but that handler wasn't being invoked because armor wasn't actually CRITICALS_VARIABLE.